README.md

```diff
@@ -74,7 +74,8 @@ This section will walk through deploying the Azure infrastructure and Ansible Au
 ### Checklist
 
 - [ ] Install this collection (or download and modify)
-- [ ] Ansible CLI tools installed locally (`ansible`)
+- [ ] Ansible CLI tools installed locally (`ansible`)  
+    *NOTE*: On Mac, use *pip* installed Ansible.  We have seen problems when trying to use *brew* installed Ansible.
```

Comment:
I don't believe this is necessary, but feel free to resolve this comment. The docs only walk through pip install https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html

Reply:
0k, let's change the format then. 
```suggestion

    **NOTE**: On Mac, use `pip` installed Ansible.  We have seen problems when trying to use `brew` installed Ansible.
```

Reply:
fixed